pkgs/top-level/python-packages.nix

```diff
@@ -22182,6 +22182,29 @@ in modules // {
     };
   };
 
+  reno = buildPythonPackage rec {
+    name = "reno-${version}";
+    version = "1.8.0";
+
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/r/reno/${name}.tar.gz";
+      sha256 = "1pqg0xzcilmyrrnpa87m11xwlvfc94a98s28z9cgddkhw27lg3ps";
+    };
+
+    # Tests require gnupg1 and git
```

Comment:
I should have been more straightforward. They are failing. I figured out it requires gpg and git, but then more errors occur along the lines of : 
```
running install tests
running test
gpg: directory '/tmp/nix-build-reno-1.8.0.drv-0/tmp0iQLQd/.gnupggpg: directory '/tmp/nix-build-reno-1.8.0.drv-0/tmpcZ19xv/.gnupg' created
' created
gpg: new configuration file '/tmp/nix-build-reno-1.8.0.drv-0/tmp0iQLQd/.gnupg/dirmngr.conf' created
gpg: new configuration file '/tmp/nix-build-reno-1.8.0.drv-0/tmp0iQLQd/.gnupg/gpg.conf' created
gpg: keybox '/tmp/nix-build-reno-1.8.0.drv-0/tmp0iQLQd/.gnupg/pubring.kbx' created
gpg: invalid item '(setpref)' in preference string
...
OS_TEST_TIMEOUT=${OS_TEST_TIMEOUT:-60} \
${PYTHON:-python} -m subunit.run discover -t ./ .  --load-list /tmp/nix-build-reno-1.8.0.drv-0/tmpHfB2Dl
======================================================================
FAIL: reno.tests.test_scanner.BasicTest.test_rename_then_delete_file
tags: worker-0
----------------------------------------------------------------------
Empty attachments:
  stderr
  stdout

pythonlogging:'': {{{
DEBUG reno.scanner scanning /tmp/nix-build-reno-1.8.0.drv-0/tmp6Mrx32/tmpy3QjPA/reporoot/releasenotes/notes (branch=None)
   DEBUG reno.scanner running git log --simplify-by-decoration --pretty="%d"
   DEBUG reno.scanner " (HEAD -> master, tag: 2.0.0)"
" (tag: 1.0.0)"
...
```
I don't know how to fix it in a reasonable amount of time.